### DISCUSSION TITLE
Volt support?

### DISCUSSION THREAD

**Discussion by misog** [nWidart/laravel-modules#1959] [General]
Volt components are not working for me, are they supported in laravel-modules?

[https://livewire.laravel.com/docs/volt](https://livewire.laravel.com/docs/volt)

**Comment by dcblogdev** [comment]
They don't appear to be, when using Livewire we rely on laravel-modules-livewire package. I've opened an issue there to see if we can have volt support.

It may be possible to manually bind the components in the server provider.

https://github.com/mhmiton/laravel-modules-livewire/issues/51

I'm going to try and see if I can manually get it working.

**Comment by dcblogdev** [comment]
This works:

So in the service provider:

```php
Volt::mount(Module::getModulePath($this->moduleName).'resources/views/livewire');
```

then have a path like `Modules/Base/resources/livewire/base/contact.php`

Then a route like:

```php
Volt::route('/contact', 'base.contact');
```

The reason to add a folder the same as the module is to avoid conflicts so you can do `module.file`
